### ISSUE TITLE
VMNet Issues

### ISSUE THREAD

**Issue by pecigonzalo** [abiosoft/colima#590]
### Description

`lima` and `colima` rely on `vmnet_socket` for some network setups. Colima installs embedded binaries to `/opt/colima` but it seems to not re-install them if already present.

This seems to be the source of the errors 
```
WARN[0016] error setting up network dependencies: error at 'preparing network': error running [/Users/someuser/.nix-profile/bin/colima daemon status default], output: "time=\"2023-01-19T11:44:43-03:00\" level=fatal msg=\"pid file not found: stat /Users/someuser/.colima/default/daemon/daemon.pid: no such file or directory\"", err: "exit status 1"  context=vm
INFO[0016] creating and starting ...                     context=vm
WARN[0016] error setting up reachable IP address: vmnet socket file not found: stat /Users/someuser/.colima/default/daemon/vmnet.sock: no such file or directory
```

present in many issue reports. After running `sudo rm -rf /opt/colima` those two issues seem to be gone.

### Version

Colima Version: 0.5.2
Lima Version: 0.13.0
Qemu Version: 0.7.2

### Operating System

- [ ] macOS Intel <= 12 (Monterrey)
- [ ] macOS Intel >= 13 (Ventura)
- [ ] macOS M1 <= 12 (Monterrey)
- [X] macOS M1 >= 13 (Ventura)
- [ ] Linux

### Output of `colima status`

INFO[0000] colima is running using QEMU
INFO[0000] arch: aarch64
INFO[0000] runtime: docker
INFO[0000] mountType: sshfs
INFO[0000] address:
INFO[0000] socket: unix:///Users/pecigonzalo/.colima/default/docker.sock

### Reproduction Steps

1.
2.
3.

**Comment by abiosoft** [comment]
The assumption is that the files were **only** created by Colima anyways and they hardly change.

I think there are two steps to a proper solution. Even though either would suffice.

1. compare the hashes of the files (not only the presence) and replace them if required.
2. provide a command for the user to reset the network settings, which will remove the files in `/opt/colima`

Thanks for the information.

**Comment by pecigonzalo** [comment]
Yeah, I did not modify them but maybe they got out of sync or something. Since I saw others report similar error messages, I thought I would be good to inform.

Both options sound fair. I think 2 should be coupled with some FAQ on when to use them.

PS: I hope this also helps clear out some of the network quirks so we can find what's going on with the assign address issues.

**Comment by snowsky** [comment]
Deleting `/opt/colima` on Mac M1 works 👍

**Comment by jubr** [comment]
Also ping here, see https://github.com/abiosoft/colima/issues/458#issuecomment-2427994367
